frontend/src/views/MessagingView.vue

```diff
@@ -90,5 +90,7 @@ export default {
 
 :deep(.v-badge__badge) {
   font-size: 0.75rem !important;
+  min-width: 18px !important;
+  padding: 2px 3px !important;
```

Comment:
!important should be avoided in css. If it's the only way it works for now we can revisit.

Reply:
Fixed, removed all !import and ensured styles still take effect. Makes me feel crazy... i only added !important because i wasn't seeing the styles take effect in the 1st place.